tests/foreman/ui/test_computeresource_libvirt.py

```diff
@@ -154,7 +154,7 @@ def test_positive_provision_end_to_end(
         location=[module_location],
         organization=[module_sca_manifest_org],
     ).create()
-    with session:
+    with sat.ui_session() as session:
```

Comment:
I think here you might need to select org an loc specifically, when you use .ui_session contextmanager